<ui version="4.0" >
 <author>Omar Andres Zapata Mesa</author>
 <class>OJohnMainWindow</class>
 <widget class="QMainWindow" name="OJohnMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OJohnGui</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QGroupBox" name="groupBox_5" >
      <property name="title" >
       <string>Archivos</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="OLoadFilesButton" >
         <property name="text" >
          <string>Cargar Archivos </string>
         </property>
         <property name="shortcut" >
          <string>Ctrl+F</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="OLoadDictionaryButton" >
         <property name="text" >
          <string>Cargar Diccionarios</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item rowspan="2" row="0" column="1" >
     <widget class="QGroupBox" name="groupBox_6" >
      <property name="title" >
       <string>CPU </string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Prioridad Usuario</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QSlider" name="OUserPriorityHorizontalSlider" >
         <property name="minimum" >
          <number>-20</number>
         </property>
         <property name="maximum" >
          <number>20</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Baja</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Normal</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Alta</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Uso de la CPU</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>Usuario</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QProgressBar" name="OCpuUserProgressBar" >
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Sistema</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QProgressBar" name="OCpuSystemProgressBar" >
         <property name="value" >
          <number>0</number>
         </property>
         <property name="alignment" >
          <set>Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Total</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" >
        <widget class="QProgressBar" name="OCpuProgressBar" >
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item rowspan="2" row="0" column="2" >
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Sesion</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="OLabelSession" >
         <property name="font" >
          <font>
           <family>OStayPuft</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>John</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="OViewLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap resource="OJohn.qrc" >:/home/andresete/WorkSpace/OHackSuite/OIcons/JOhnICons/BigO.png</pixmap>
         </property>
         <property name="scaledContents" >
          <bool>true</bool>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="title" >
       <string>Opciones Basicas</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="font" >
          <font>
           <family>OStayPuft</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Modo</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2" >
        <widget class="QLabel" name="label_2" >
         <property name="font" >
          <font>
           <family>OStayPuft</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Tipo de Encriptaci√≥n</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="OSingularCheckBox" >
         <property name="text" >
          <string>Singular</string>
         </property>
         <property name="icon" >
          <iconset resource="OJohn.qrc" >:/home/andresete/WorkSpace/OHackSuite/OIcons/JOhnICons/S.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2" >
        <widget class="QComboBox" name="OSaltsComboBox" >
         <item>
          <property name="text" >
           <string>Cualquiera</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>des</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>bsdi</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>md5</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>bf</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>afs</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>lm</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QCheckBox" name="OIncrementalCheckBox" >
         <property name="text" >
          <string>Incremental</string>
         </property>
         <property name="icon" >
          <iconset resource="OJohn.qrc" >:/home/andresete/WorkSpace/OHackSuite/OIcons/JOhnICons/I.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="OIncrementalModeComboBox" >
         <item>
          <property name="text" >
           <string>all</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>alnum</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>digits</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>laman</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>alpha</string>
          </property>
         </item>
        </widget>
       </item>
       <item rowspan="2" row="2" column="2" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="font" >
          <font>
           <family>OStayPuft</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Salvar Nivel de Memoria</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="OExternalCheckBox" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Externo</string>
         </property>
         <property name="icon" >
          <iconset resource="OJohn.qrc" >:/home/andresete/WorkSpace/OHackSuite/OIcons/JOhnICons/E.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QComboBox" name="OExternalModeComboBox" >
         <item>
          <property name="text" >
           <string>Filter_Alpha</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Filter_Digits</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Filter_Alnum</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Filter_LanMan</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>LanMan</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Double</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Parallel</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Strip</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Keyboard</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QSlider" name="OSaveMemoryHorizontalSlider" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>3</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance" >
          <bool>false</bool>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="4" column="3" >
        <widget class="QLCDNumber" name="lcdNumber" >
         <property name="sizeIncrement" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="numDigits" >
          <number>1</number>
         </property>
         <property name="mode" >
          <enum>QLCDNumber::Dec</enum>
         </property>
         <property name="segmentStyle" >
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="value" stdset="0" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Informaci√≥n</string>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="OShowPushButton" >
         <property name="text" >
          <string>Passwords</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ODataPushButton" >
         <property name="text" >
          <string>Datos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>Limpiar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2" >
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Ejecuci√≥n</string>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="OStopPushButton" >
         <property name="text" >
          <string>Parar</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>28</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ORunPushButton" >
         <property name="text" >
          <string>Ejecutar</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>43</width>
           <height>16</height>
          </size>
         </property>
         <property name="shortcut" >
          <string>Ctrl+R</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3" >
     <widget class="QDockWidget" name="ODockWidget" >
      <property name="font" >
       <font>
        <family>OStayPuft</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="floating" >
       <bool>false</bool>
      </property>
      <property name="features" >
       <set>QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas" >
       <set>Qt::LeftDockWidgetArea|Qt::NoDockWidgetArea|Qt::RightDockWidgetArea</set>
      </property>
      <property name="windowTitle" >
       <string>Mensages Log</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QTextEdit" name="OLogPanelTextEdit" >
          <property name="font" >
           <font>
            <family>Bitstream Vera Sans</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda" >
    <property name="title" >
     <string>Ayuda</string>
    </property>
    <addaction name="OActionHelp" />
    <addaction name="separator" />
    <addaction name="OActionCredits" />
   </widget>
   <widget class="QMenu" name="menuInicio" >
    <property name="title" >
     <string>Inicio</string>
    </property>
    <addaction name="OActionNewSession" />
    <addaction name="OActionLoadSession" />
    <addaction name="separator" />
    <addaction name="OActionQuit" />
    <addaction name="separator" />
   </widget>
   <addaction name="menuInicio" />
   <addaction name="menuAyuda" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="OToolBar" >
   <property name="windowModality" >
    <enum>Qt::NonModal</enum>
   </property>
   <property name="windowTitle" >
    <string>OJohnToolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="separator" />
  </widget>
  <action name="actionNueva_Sesion" >
   <property name="text" >
    <string>Nueva Sesion</string>
   </property>
  </action>
  <action name="OActionLoadSession" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Cargar Sesion</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="OActionQuit" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Salir</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="OActionHelp" >
   <property name="text" >
    <string>Ayuda</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="OActionCredits" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Creditos</string>
   </property>
  </action>
  <action name="actionConfiuracion" >
   <property name="text" >
    <string>Confiuracion</string>
   </property>
  </action>
  <action name="actionNueva_Sesion_2" >
   <property name="text" >
    <string>Nueva Sesion</string>
   </property>
  </action>
  <action name="OActionNewSession" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="OJohn.qrc" >:/home/andresete/WorkSpace/OHackSuite/OIcons/JOhnICons/N.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Nueva Sesion</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="OJohn.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>OSaveMemoryHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>373</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>408</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>OLogPanelTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>408</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
